python-threatexchange/threatexchange/exchanges/impl/techagainstterrorism_api.py

```diff
@@ -90,27 +96,38 @@ def get_client(self) -> api.TATHashListAPI:
     def fetch_iter(
         self,
         _supported_signal_types: t.Sequence[t.Type[SignalType]],
-        checkpoint: t.Optional[state.TFetchCheckpoint],
+        checkpoint: t.Optional[TATCheckpoint],
     ) -> t.Iterator[_TypedDelta]:
 
         client = self.get_client()
-        result = client.get_hash_list()
 
-        translated = (_get_delta_mapping(entry) for entry in result)
+        start_time = ""
+
+        if checkpoint is not None:
+            start_time = checkpoint.checkpoint
 
-        yield state.FetchDelta(
-            dict(t for t in translated if t[0][0]),
-            checkpoint=state.NoCheckpointing(),
-        )
+        for result in client.fetch_hashes_iter(start_time):
```

Comment:
blocking; This isn't a start time, it's a checkpoint, right?
```suggestion
        checkpoint = "" if checkpoint is None else checkpoint.checkpoint
        for result in client.fetch_hashes_iter(checkpoint):
```